### ISSUE TITLE
Configurable width/height for each chart

### ISSUE THREAD

**Issue by stufro** [stufro/obsidian-hill-charts#3]
Make it possible to override the chart width and height in the input yml. e.g.

```yml
points:
  - position: 50
    text: Foo
width: 500
height: 300
```

**Comment by rmateu-pricesmart** [comment]
+1 on this request. But having so much fun with this extension, thanks for making it.

**Comment by stufro** [comment]
Thanks for your kind words - I've just released version 0.2.0 which includes this feature, enjoy.
